src/orchard/inspect.clj

```diff
@@ -379,6 +393,32 @@
         (render-map-values (meta obj)))
     inspector))
 
+;; Datafy, see https://corfield.org/blog/2018/12/03/datafy-nav
+
+(defn- datafy-nav-data [obj]
+  (let [data (datafy obj)]
+    (if (map? data)
+      (into {} (for [k (keys data)
+                     :let [v (get data k)
+                           obj' (nav obj k v)]
+                     :when obj']
+                 [k (datafy obj')]))
+      data)))
+
+(declare inspect)
+
+(defn- render-datafy [inspector obj]
+  (if datafy-supported?
```

Comment:
Please use `if-not`, placing the smallest conditional branch first

(it's a poor man's https://wiki.c2.com/?GuardClause)

Reply:
Fixed